examples/console/main.go

```diff
@@ -283,6 +291,28 @@ func prompt(agent *agentpkg.Agent, horizonClient horizonclient.ClientInterface,
 		return agent.DeclareClose()
 	case "close":
 		return agent.Close()
+	case "declarecloseidx":
+		idx, err := strconv.Atoi(params[1])
+		if err != nil {
+			return err
+		}
+		tx := closeAgreements[idx].SignedTransactions().Declaration
```

Comment:
💡 if idx is too large this will error, thoughts on throwing an error like `"given idx is too large. Please pass value smaller than %d," len(closeAgreements)`

Reply:
+1 9a458fd. I also added another command `listagreements` that will list the agreements so it is easier to figure out which one to submit.